package.json

```diff
@@ -7,32 +7,25 @@
   "author": "Czysty <lukasz.czyszczonik@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "babel index.js -d dist",
+    "dev": "nodemon example/index.js",
```

Comment:
Are you missing the `nodemon` in `devDependencies`?

Reply:
Oh yes, added now :)